### ISSUE TITLE
Issue with Exporter and Iron Furnaces

### ISSUE THREAD

**Issue by Rithenar** [CyclopsMC/IntegratedDynamics#1229]
#### Issue type:

- :snail: Performance issue 

____

#### Short description:

I have an exporter with a variable card that points to an inventory reader with all items I need to be exported to an Obsidian Furnace from the Iron Furnaces mod. When I put the card in the Export Items position the lag from the furnace spikes up to around to about 
115,671 us/t. Without the variable card, it goes down to ~ 220 us/t.

I changed the variable to one programmed to a list of items instead of the inventory reader and I had the same exact issue.
#### Steps to reproduce the problem:

1. ... Put items to burn in Obsidian Furnace from Iron Furnaces - they burn fast!
2.  Put variable card pointing at inventory reader with 11 items, lag spikes and the furnace stutters while cooking
3- Remove variable card, open furnace, things are lightning fast again
4- Put variable card with same 11 items programed in programmer using list - furnace goes back to stuttering and lag spikes again
5- Remove variable card, everything works well
____

#### Versions:

- This mod: 1.18.2-1.11.12
- Minecraft: 1.18.2
- Forge: 40.1.80

#### Profiler output:

![Screenshot 2022-11-02 144147](https://user-images.githubusercontent.com/9939332/199598075-9f0d69aa-a995-4fad-bd96-edcd82073e7b.png)
![Screenshot 2022-10-28 144346](https://user-images.githubusercontent.com/9939332/199598164-a0971c3d-365e-4302-b333-47861bf5cb31.png)

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
Could you share a profiling log, as explained [here](https://github.com/CyclopsMC/CyclopsCore/blob/master-1.18/.github/CONTRIBUTING.md#performance-issues-snail)?

**Comment by Rithenar** [comment]
Sure, here you go https://spark.lucko.me/c3Lmarzz7N

<blockquote><img src="https://spark.lucko.me/thumb/c3Lmarzz7N.png" width="48" align="right"><div><strong><a href="https://spark.lucko.me/">spark</a></strong></div><div>spark is a performance profiler for Minecraft clients, servers, and proxies.</div></blockquote>

**Comment by rubensworks** [comment]
Thanks!
Could you share a screenshot of your setup?
Because I can't really see the connection of the profiling log and your description.

**Comment by Rithenar** [comment]
It's quite simple. I already removed it because of the lag so I just recreated it the way I had it before.
![Screenshot 2022-11-03 081913](https://user-images.githubusercontent.com/9939332/199731278-54d64d36-be91-4c19-9c52-6eeb2c114cf6.png)
![Screenshot 2022-11-03 081907](https://user-images.githubusercontent.com/9939332/199731279-252020f2-00ca-41d6-bf3e-5a5ad6ca4a25.png)

Let me know if you are looking for something different.

BTW, the same happened with a Pulverizer from Thermal Series. Exact same setup
